front/lib/resources/space_resource.ts

```diff
@@ -605,7 +605,7 @@ export class SpaceResource extends BaseResource<SpaceModel> {
     return [
       {
         workspaceId: this.workspaceId,
-        roles: [{ role: "admin", permissions: ["admin", "write"] }],
```

Comment:
This is awesome but might break a few things?

Let's test thoroughly and give it a spin 🦾

Reply:
Actually we had some permission checks on "write" instead of "admin" to administrate the space and edit views. 
I changed the check from canWrite to canAdministrate in these cases. Builders should not be able to edit data source views.